### DISCUSSION TITLE
Trigger on new row suddenly not working - any idea why?

### DISCUSSION THREAD

**Discussion by code-for-asia** [Budibase/budibase#10944] [Help]
Hello all, I was very excited to start building on Budibase since Friday night and managed to achieve all desired functionalities until my automations that were supposed to trigger on new row suddenly stopped working. When I run a test to send email and Discord notifications, it works; but when it's supposed to trigger based on new row which was working previously, it stopped working from around 2.06pm (GMT+8) on 18 June.

![image](https://github.com/Budibase/budibase/assets/89963631/1c675193-70e8-40dc-a5b6-0eab6ebd0fc1)

I tried to create new forms, even new apps using Budibase database instead of the MySQL server I was used; yet the automations continue to not work or get triggered. Also tried to make it trigger by calling it after user submits, still won't work.

Is anyone else encountering this issue? Might I have ran out of free automations? Is there anyway to keep track of this?

**Comment by ducnhoxlatui** [comment]
Hmmm, maybe that i used to be in this situation and these are my view:
- Has your app been published yet? Because automation can only be actived when app is publishing (test in automation is just the test at that time so you need to publish app after testing).
- If you are using free acount, please check the limitation of automation (because it only has 200 times).
- You can also use another type of automation which is called Cron trigger. Set the time to check your DB and maybe that you can create another column in your table such as IsSend to check send or not.
